docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -1,199 +1,376 @@
-## BASIC01: The domain must have a parent domain
+# BASIC01: Check for the parent zone and the zone itself
 
-### Test case identifier
+## Test case identifier
 **BASIC01**
 
-### Objective
 
-In order for a domain (zone) to work, it must be delegated from a 
-zone higher up in the DNS hierarchy (a parent domain/zone). 
-This Test Case will determine if parent and child zones exist.
+## Table of contents
 
-If the zone to be tested is the root zone, it has no parent or
-delegation and will always pass this Test Case.
-
-If no parent can be determined, there cannot be any delegation.
-
-If the child zone does not exist (is not delegated), the only 
-test case to be run after this test case is BASIC03. However, 
-if the test type is an undelegated test, then all other test cases 
-can be run even if the child zone is not delegated.
-
-### Inputs
-
-Input for this Test Case:
-* "Child Zone" - The label of the domain name (zone) to be tested
-* "Root Name Servers" - The IANA [List of Root Servers] 
-* "Test Type" - The test type with values "undelegated test" or 
-  "normal test".
+* [Objective](#Objective)
+* [Scope](#Scope)
+* [Inputs](#Inputs)
+* [Summary](#Summary)
+* [Test procedure](#Test-procedure)
+* [Outcome(s)](#Outcomes)
+* [Special procedural requirements](#Special-procedural-requirements)
+* [Intercase dependencies](#Intercase-dependencies)
+* [Terminology](#terminology)
```

Comment:
Could you lower case the anchors? See https://github.com/zonemaster/zonemaster/blob/master/docs/internal-documentation/maintenance/StyleGuide.md

Reply:
Fixed. This PR was done long time ago...